.github/workflows/coverage-report.yml

```diff
@@ -1,7 +1,7 @@
 name: Regression Tests Coverage Report
 
 on:
-  pull_request_target:
+  pull_request:
```

Comment:
Without the `pull_request_target`, I don't think it has access to the Secret needed to post/update the comment

Reply:
I see, of course, oh well -- I'll drop that commit.

Reply:
If you want to test it, you can push the updated Yaml to your own master branch in your fork (or change your default branch to this branch) and open a PR on your repo. You'd need to to add your own PAT as a GitHub secret too

Reply:
Thanks so much for detailing I'm sure I'll want to test the actions like this in the future!

Reply:
Or if you don't want to over-think it, like appartently did, just edit the bot comment after it's created/updated and remove the backticks manually 😆

Reply:
hahaha omg why didn't we think of that :eyes: